### ISSUE TITLE
Mismatched diacritics and dashes

### ISSUE THREAD

**Issue by robinwhittleton** [bob-reus/joris-karl-huysmans_against-the-grain#1]
`se find-mismatched-dashes .` and `se find-mismatched-diacritics .` throw up some potential places to normalise stuff. Can you review and make any necessary changes?

**Comment by bob-reus** [comment]
**old-time**      (2)   oldtime      (1)  
  plain-song    (1)   **plainsong**    (5)  
  **sugar-sweet**   (1)   sugarsweet   (1)  
  title-page    (1)   titlepage    (4)  

I fixed the first three (to the word in bold). Title-page is an interesting one: Merriam-Webster gives 'title page' as the correct spelling. The four instances that are spelled 'titlepage' are in titlepage.xhtml, uncopyright.xhtml and the ToC (which derives it from titlepage.xhtml, I guess). Has this ever come up before?

  bénédictine   (3)   benedictine   (2)   
  capé          (1)   cape          (3)   
  curé          (1)   cure          (9)   
  dîner         (1)   diner         (1)   
  obscène       (1)   obscene       (2)   
  pâle          (1)   pale          (19)  
  **séverin**       (1)   severin       (1)   
  sévère        (1)   severe        (3)   

As for mismatched-diacritics, I changed an instance of Severin to Séverin. The other matches are usually a case of different languages, i.e. the English obscene and the French obscène, Bénédictine for the liquor and Benedictine for the monks.
